python/tank/deploy/tank_command.py

```diff
@@ -47,9 +47,9 @@
 ###############################################################################################
 # Built in actions (all in the tank_commands sub module)
 
-BUILT_IN_ACTIONS = [setup_project.SetupProjectAction, 
+BUILT_IN_ACTIONS = [setup_project.SetupProjectAction,
                     setup_project_wizard.SetupProjectFactoryAction,
-                    core_upgrade.CoreUpgradeAction, 
+                    core_upgrade.CoreUpdateAction,
```

Comment:
Why the class rename?  I get that it is slightly more accurate, but this is part of our public API so we shouldn't change it without good reason and ideally backward compatibility.  I think there are people out there who deal with things like core updates programmatically and I think this would break their code.